.npmignore

```diff
@@ -0,0 +1,3 @@
+scripts
```

Comment:
Not sure about this. Wasn't aware of what the script is doing until looking at this now. Seems it is used here: https://github.com/ember-fastboot/ember-cli-fastboot/blob/8a528fe076372f3c03f0a2c673999d87a013b580/package.json#L19. Excluding the file from the package would break this, wouldn't it?